tests/Doctrine/Tests/DBAL/Functional/Schema/PostgreSqlSchemaManagerTest.php

```diff
@@ -42,7 +42,7 @@ public function testGetSchemaNames()
         $names = $this->_sm->getSchemaNames();
 
         self::assertInternalType('array', $names);
-        self::assertTrue(count($names) > 0);
+        self::assertGreaterThan(0, $names);
```

Comment:
`count($names)` should be kept

Reply:
Done!

Reply:
Given that the previous assertion guarantees it's an array, can we subsequently use `self::assertNotEmpty($names)`?